lib/OpenQA/Utils.pm

```diff
@@ -34,8 +34,46 @@ $Data::Dumper::Terse = 1;
 
 my $FRAG_REGEX = FRAGMENT_REGEX;
 
+my (%BUGREFS, %BUGURLS, $MARKER_REFS, $MARKER_URLS);
+BEGIN {
```

Comment:
this begin thingy looks a bit ugly though

Reply:
Agreed, no way to avoid it though, constants can only be created at compile time, which requires a `BEGIN` block.